src/app.ts

```diff
@@ -32,8 +32,20 @@ export default (async () => {
     const appFactory = new AppFactory({ module, client });
 
     appFactory.createApp();
+    await client.initialize();
 
-    client.initialize();
+    const guild = client.guilds.cache.get(process.env.GUILDID || "");
+    if (guild) {
+      const channels = await guild.channels.fetch();
+      for (const channel of channels.values()) {
+        if (channel && channel.type === ChannelType.GuildVoice) {
+          for (const [memberId, member] of channel.members) {
+            const isAFK = guild.afkChannelId === channel.id;
+            client.voiceManager.addUserToCollection(memberId, isAFK);
+          }
+        }
+      }
+    }
```

Comment:
Okay. This _should_ do the trick, at least it worked for me in test server.
Don't think that the single server fetching would be a problem. Up to debate.